recipes/pyside/bld.bat

```diff
@@ -0,0 +1,43 @@
+rem  @echo off
+
+set CMAKE_CONFIG="Release"
+
+cd %SRC_DIR%\sources\shiboken
+
+mkdir build_win
+cd build_win
+
+set PYTHON_LIBRARY=%PREFIX%\libs\python%PY_VER:~0,1%%PY_VER:~2,1%.lib
+
+cmake -LAH -G"NMake Makefiles"                               ^
+    -DCMAKE_FIND_ROOT_PATH="%LIBRARY_PREFIX%"                ^
+    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"                ^
+    -DPYTHON_SITE_PACKAGES="%SP_DIR%"                        ^
+    -DCMAKE_BUILD_TYPE=Release                               ^
+    -DBUILD_TESTS=OFF                                        ^
+    -DUSE_PYTHON3=%PY3K%                                     ^
+    -DPYTHON3_EXECUTABLE="%PYTHON%"                          ^
+    -DPYTHON3_INCLUDE_DIR="%PREFIX%\include"                 ^
+    -DPYTHON3_LIBRARY="%PYTHON_LIBRARY%"                     ^
+    ..
+if errorlevel 1 exit 1
+
+cmake --build . --config %CMAKE_CONFIG% --target INSTALL
```

Comment:
PySide is known for being buggy (that's why we don't support it anymore in Spyder 3.0), but people still want to use it because of its liberal license.

So we shouldn't demand @jschueller to fix these bugs :-) However, @jschueller please add a comment about why `ctest` is not activated.